### PR TITLE
Standardize on exclaiming about committing suicide.

### PR DESCRIPTION
**Pull request by FunnyMan3595**
## What Does This PR Do
s/trying to commit suicide\\./trying to commit suicide!/
(plus a few manual ones that use a word other than 'suicide')

## Why It's Good For The Game
Consistency is good.

## Testing
It compiled.

## Changelog
:cl:
spellcheck: All suicide actions now put an exclamation point on "trying to commit suicide!"
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/kitchen.dm [hunk 1]
```diff
@@ -126,8 +126,8 @@
 	AddComponent(/datum/component/surgery_initiator/robo)
 
 /obj/item/kitchen/knife/suicide_act(mob/user)
-	user.visible_message(pick("<span class='suicide'>[user] is slitting [user.p_their()] wrists with [src]! It looks like [user.p_theyre()] trying to commit suicide.</span>", \
-						"<span class='suicide'>[user] is slitting [user.p_their()] throat with [src]! It looks like [user.p_theyre()] trying to commit suicide.</span>", \
+	user.visible_message(pick("<span class='suicide'>[user] is slitting [user.p_their()] wrists with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>", \
+						"<span class='suicide'>[user] is slitting [user.p_their()] throat with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>", \
 						"<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku.</span>"))
```
Comment (Henri215) [inline code/game/objects/items/weapons/kitchen.dm:131]:
```suggestion
						"<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku!</span>"))
```

File: code/game/objects/items/weapons/melee/energy_melee_weapons.dm [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 /obj/item/melee/energy/suicide_act(mob/user)
 	user.visible_message(pick("<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku.</span>", \
```
Comment (Henri215) [inline code/game/objects/items/weapons/melee/energy_melee_weapons.dm:44]:
```suggestion
	user.visible_message(pick("<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku!</span>", \
```

File: code/modules/surgery/tools.dm [hunk 3]
```diff
@@ -125,8 +125,8 @@
 
 
 /obj/item/scalpel/suicide_act(mob/user)
-	to_chat(viewers(user), pick("<span class='suicide'>[user] is slitting [user.p_their()] wrists with [src]! It looks like [user.p_theyre()] trying to commit suicide.</span>",
-						"<span class='suicide'>[user] is slitting [user.p_their()] throat with [src]! It looks like [user.p_theyre()] trying to commit suicide.</span>",
+	to_chat(viewers(user), pick("<span class='suicide'>[user] is slitting [user.p_their()] wrists with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>",
+						"<span class='suicide'>[user] is slitting [user.p_their()] throat with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>",
 						"<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku.</span>"))
```
Comment (Henri215) [inline code/modules/surgery/tools.dm:130]:
```suggestion
						"<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku!</span>"))
```

**Review by Henri215 — COMMENTED** [review]
There is another seppuku on weaponry.dm

**Review by Henri215 — APPROVED** [review]
Looks good to me!

**Review by S34NW — APPROVED** [review]

**Pull request merged**